ProjectSettings/QualitySettings.asset

```diff
@@ -29,6 +29,12 @@ QualitySettings:
     vSyncCount: 0
     lodBias: 0.3
     maximumLODLevel: 0
+    streamingMipmapsActive: 0
```

Comment:
Best to revert these .asset changes as they aren't part of this PR

Reply:
I'll add this in it's own or